.github/workflows/build-dependencies.txt

```diff
@@ -1,18 +1,11 @@
-acl
 alien
-attr
 autoconf
-bc
+automake
 build-essential
-curl
-dbench
+dkms
```

Comment:
We don't use dkms now, looks like we don't need to add it.

Reply:
I will try this.